src/effects_1.c

```diff
@@ -0,0 +1,2093 @@
+#include "global.h"
```

Comment:
Will these files eventually contain code? If not they should go into src/data

Reply:
yes they will